src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  font-style: normal;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.label::before {
+  position: absolute;
+  content: "";
+}
+
+.label-big::before {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
the same for `-big` and `-small` - just move it to a `label::before`

Reply:
also, the correct way is to create a common `label` with all styles, and `label--small` with overwrite styles, [checklist 2](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)